src/components/Card/ProductCard/ProductCard.jsx

```diff
@@ -1,15 +1,20 @@
+import { Link } from "react-router-dom";
+import { useFilter } from "../../../contexts/filterContext/context";
 import styles from "./ProductCard.module.css";
 export const ProductCard = (props) => {
   const { img, name, price, description } = props;
```

Comment:
could have de-structured in the signature itself.
If you've done it for better readability, then it's fine

Reply:
Yeah I'm passing the remaining props to Product Details page so did that way